README.md

```diff
@@ -55,6 +55,51 @@ console.log('from Base64 String: ', scteMessageFromBase64String(B64_STRING));
 console.log('from Hex String: ', scteMessageFromHexString(HEX_STRING));
 ```
 
+**Android**
+
+## Usage
+
+- In build.gradle under dependencies add the following
+```
+ implementation 'com.realeyes:scte35decoder:0.100.2'
+ 
+```
+- Add environmental variables for your Github token and username credentials
+### Step 1 : Generate a Personal Access Token for GitHub
+- Inside you GitHub account:
+	- Settings -> Developer Settings -> Personal Access Tokens -> Generate new token
+	- Make sure you select the following scopes (" write:packages", " read:packages") and Generate a token
+	- After Generating make sure to copy your new personal access token. You won’t be able to see it again!
+
+### Step 2: Add your GitHub - Personal Access Token details to environment variables
+	
+- You add the **GPR_USER** and **GPR_API_KEY** values to your environment variables on you local machine or build server to avoid creating a github properties file
+- Then under the test package you can find ExampleUnitTest which uses the library methods and it will run successfully
```

Comment:
In this README and in scte35-android aren't we missing Step 3? gradle configuration?
https://github.com/enefce/AndroidLibraryForGitHubPackagesDemo#step-3--update-buildgradle-inside-the-application-module

https://github.com/realeyes-media/scte35-android/blob/main/app/build.gradle#L28

Reply:
Okay would be better to add them , i added this section here already now